src/Mafia/Init.hs

```diff
@@ -150,18 +154,16 @@ data Profiling =
   | EnableProfiling
     deriving (Eq, Ord, Show)
 
-data InstallState =
-  InstallState {
-      isSourceDependencies :: Set SourcePackage
-    , _isCabalFile         :: Hash
-    } deriving (Eq, Ord, Show)
-
 data MafiaState =
   MafiaState {
-      msInstallState :: InstallState
```

Comment:
whether profiling is installed is morally part of the install state, which is just the mafia state now

bumping the version on the json state is basically free, recreating it only takes 5-10secs